src/main/java/org/topbraid/shacl/tools/AbstractTool.java

```diff
@@ -67,6 +69,15 @@ class AbstractTool {
 		spec.setDocumentManager(dm);
 	}
 	
+	protected int getMaxIterations(String[] args) {
+		for(int i = 0; i < args.length - 1; i++) {
+			if(MAX_ITERATIONS.equals(args[i])) {
+				return Integer.parseInt(args[i + 1]);
+			}
+		}
+		System.err.println("Missing -maxiterations, e.g.: -maxiterations 100. Defaulting to 1.");
```

Comment:
This is an optional argument so I don't think it should print anything if it's missing, esp not to System.err.

Reply:
Got it!